domains/mraxays.json

```
@@ -5,6 +5,11 @@
   },
   "records": {
     "A": ["216.150.1.1"],
-    "TXT": "peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"
+    "TXT": ["peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"]
+  },
+  "subdomains": {
+    "_vercel": {
+      "TXT": ["vc-domain-verify=mraxays.is-a.dev,6b8700111c94b97c2473"]
+    }
```

subdomains is not supported, you need to create another json file e.g. _vercel.yourname.json and add the TXT records there
```suggestion
  }
```